src/main/java/org/terasology/launcher/LauncherInitTask.java

```diff
@@ -68,6 +69,8 @@ protected LauncherConfiguration call() {
             // launcher settings
             final BaseLauncherSettings launcherSettings = getLauncherSettings(launcherDirectory);
 
+            new LauncherSettingsValidator().validate(launcherSettings);
```

Comment:
Can `validate(...)` be implemented as a static method? I don't see why in this setup the class needs to be instantiated, as it is a) always the same, and b) only used once. Either we create the rules here and pass them to hte validator instance, or make the validator itself static.

Reply:
Yes, I think it'll make more sense to turn `validate()` static and also have a static list of rules.